app/services/dttp/contact_update.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Dttp
+  class ContactUpdate
+    include ServicePattern
+
+    class Error < StandardError; end
+
+    attr_reader :trainee, :trainee_creator_dttp_id, :batch_request
+
+    def initialize(trainee:, trainee_creator_dttp_id:)
+      @trainee = Dttp::TraineePresenter.new(trainee: trainee)
```

Comment:
Can we name this `@trainee_presenter` so as it's clearer that we're dealing with a `TraineePresenter` rather than a `Trainee`?